src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,14 @@
+package duke.command;
+
+import duke.utils.DukeException;
+import duke.utils.DukeFileHandler;
+import duke.utils.TaskList;
+import duke.utils.Ui;
+
+
+/**
+ * Corresponds to commands the users wants to execute.
+ */
+public abstract class Command {
+    public abstract void execute(TaskList tasks, Ui ui, DukeFileHandler fileHandler) throws DukeException;
```

Comment:
It seems you may have missed out on adding JavaDoc comments for the following method. Do remember to add them in when fine-tuning your file! 👍🏻